src/commands/game_options/goal.ts

```diff
@@ -66,88 +69,166 @@ export default class GoalCommand implements BaseCommand {
         priority: 120,
     });
 
+    slashCommands = (): Array<Eris.ChatInputApplicationCommandStructure> => [
+        {
+            name: "goal",
+            description: LocalizationManager.localizer.translate(
+                LocaleType.EN,
+                "command.goal.interaction.description"
+            ),
+            type: Eris.Constants.ApplicationCommandTypes.CHAT_INPUT,
+            options: [
+                {
+                    name: "goal",
+                    description: LocalizationManager.localizer.translate(
+                        LocaleType.EN,
+                        "command.goal.interaction.description"
+                    ),
+                    type: Eris.Constants.ApplicationCommandOptionTypes.INTEGER,
+                    required: true,
```

Comment:
Should we make it disable the goal if no args are passed? Which would mean `required: false`